src/time_zone_info.cc

```diff
@@ -363,7 +363,7 @@ bool TimeZoneInfo::ExtendTransitions() {
       if (last_time < ta->unix_time) transitions_.push_back(*ta);
       transitions_.push_back(*tb);
     }
-    if (last_year_ == limit) break;
+    if (last_year_ > limit) break;
```

Comment:
Just checking: should the reservation on line 341 be adjusted?

Reply:
Yes.  Good catch.

And that led me to a better explanation and implementation, which I'll send in a new PR.

Thanks.